packages/react/src/primitives/types/phoneNumberField.ts

```diff
@@ -3,24 +3,20 @@ import * as React from 'react';
 import { SelectFieldProps } from './selectField';
 import { TextInputFieldProps } from './textField';
 
-export interface PhoneNumberFieldProps extends TextInputFieldProps {
+interface PartialPhoneNumberField
```

Comment:
`Partial` is a utility type in Typsecript, prefer avoiding using it in type names